src/ert/scheduler/lsf_driver.py

```diff
@@ -103,6 +103,21 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def get_missing_jobids_from_bjobs_output(
```

Comment:
There should be something in the function name to indicate that this is filtering jobs based on submission time.